app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -141,7 +128,10 @@ class OptionsFragmentTest {
 
   @Before
   fun setUp() {
-    TestModule.forceEnableLanguageSelectionUi = true
+    TestPlatformParameterModule.forceEnableLanguageSelectionUi(true)
+    TestPlatformParameterModule.forceEnableEditAccountsOptionsUi(
+      ENABLE_EDIT_ACCOUNTS_OPTIONS_UI_DEFAULT_VALUE
```

Comment:
Perhaps this should be fixed to be the same value everytime?

Reply:
removing edit accounts force method since it isn't needed